aws/src/integration/java/org/apache/iceberg/aws/s3/signer/TestS3RestSigner.java

```diff
@@ -122,9 +123,8 @@ public static void afterClass() throws Exception {
     // there aren't other token refreshes being scheduled after every sign request and after
     // TestS3RestSigner completes all tests, there should be only this single token in the queue
     // that is scheduled for refresh
-    assertThat(validatingSigner.icebergSigner)
-        .extracting("authManager")
-        .extracting("refreshExecutor")
+    assertThat(ThreadPools.authRefreshPool())
```

Comment:
I think we should roll back these test changes here and in the other test class, since nothing really changed in terms of the implementation